app/services/users/profile.rb

```diff
@@ -148,7 +148,7 @@ def veteran_status
         served_in_military: user.served_in_military?
       }
     rescue => e
-      scaffold.errors << Users::ExceptionHandler.new(e, 'EMIS').serialize_error
+      scaffold.errors << Users::ExceptionHandler.new(e, 'Vet360').serialize_error
```

Comment:
I _think_ this was the culprit of sign on failing in staging when I changed it in my original PR: https://github.com/department-of-veterans-affairs/vets-api/pull/15320/files#diff-64abab86a25a2db0f3b93876d46a4098b13d0a5b689da27c8171c2edb9cd88e5

Will test on Monday.

Reply:
I tested this locally and it failed. I discovered I need an accompanying change in vets-website. Once I had that in place locally, testing was successful 🎉 I'll put up that PR and link it in the description.

Reply:
Vets-website PR: https://github.com/department-of-veterans-affairs/vets-website/pull/27851

Reply:
Doesn't this vets-website PR need to be merged before this PR?

Reply:
I wasn't actually sure what the order needed to be, but that sounds good to me. ~I am planning to merge them at the same time, but~ I'll do the vets-website one first. 

OH... I think I see what you're saying, @bosawt. Since vets-website and vets-api don't finish deploying at _exactly_ the same time, you're saying the same error that was seen in staging would be seen in prod 😟  for the time in between when the deploys finish? For example, say vets-api finishes deploying at 3:10 but vets-website doesn't finish until 3:40, sign-in would be broken in prod (for some users) for 30 min. 

So I'll want to have the vets-website PR deployed and _in production_ before this PR is deployed to prod

Reply:
I've modified the vets-website PR to handle emis and va profile as the external services. That way, the vets-website code can deploy one day and vets-website the next.